plugin/src/org/checkerframework/languageserver/eclipse/LanguageServerStreamConnectionProvider.java

```diff
@@ -118,6 +118,13 @@ public LanguageServerStreamConnectionProvider() {
       commands.add(commandOptions);
     }
 
+    
+    commands.add("--checkers");
+    String[] checkers = typeChecker.split("\\,");
```

Comment:
Line 114 above already adds the checkers. Should they really be separated by spaces here? Having the flag twice seems like an error.

Reply:
Sorry I was wrong. Checkers' names are separated by commas.
https://checkerframework.org/manual/#shorthand-for-checkers